mongodb/client.bal

```diff
@@ -49,8 +43,7 @@ public isolated client class Client {
                 }
             }
         }
-        self.datasource = check initClient(config);
```

Comment:
Shall we describe the usecase of using databaseName when we initailiaze?

Reply:
Fixed: https://github.com/ballerina-platform/module-ballerinax-mongodb/pull/146/commits/688282aeb3fe7eb0322b6512687e3e0dc003a9ad#diff-fc5ddf96bf9bd4eb5772667ec37d3ed50a3593b46b65688964828652ff96e3ecR32